website/blog/2023-01-12-version-071.md

```diff
@@ -0,0 +1,246 @@
+---
+title: 'React Native 0.71: TypeScript by Default, Flexbox Gap, and more...'
+authors: [lunaleaps, mattcarrollcode, cortinico, kelset]
+tags: [announcement, release]
+date: 2023-01-12
+---
+
+Today we’re releasing React Native version 0.71. This is a feature-packed release which includes TypeScript by default, Flexbox gap support, new web-inspired props, developer experience improvements, and improvement to the New Architecture experimental experience.
```

Comment:
> New Architecture experimental experience

This doesn't really roll off the tongue. I know there was a bunch of debate about "experimental", so if you want to include all those words, how about "experimental New Architecture experience"

Reply:
Updated, what do you think?